assets/src/gql/payments.graphql

```diff
@@ -9,6 +9,7 @@ enum CardTypeBrand {
   discover
   jcb
   unionpay
+  maestro
```

Comment:
is this an old file? I thought we wouldn't need this stuff defined on this side anymore since moving to server graphql @jonathaningram  ?

Reply:
@dagoof yeah not sure, happy to remove it if it's useless now. @acodegirl you can probably check to see if this file is being used. if not it can be deleted and codegen re-run.